python/cudf_polars/pyproject.toml

```diff
@@ -137,6 +139,10 @@ fixable = ["ALL"]
 
 [tool.ruff.lint.per-file-ignores]
 "**/tests/**/*.py" = ["D"]
+"**/cudf_polars/typing/__init__.py" = [
+  # pyupgrade
+  "UP007", # Use `X | Y` for type annotations
+]
```

Comment:
This recommendation is purely about style (not safety, efficiency, etc.), and I really think a `Union[` with wrapping is easier to read for long union types like the ones `ruff` wanted to change here, e.g.:

https://github.com/rapidsai/cudf/blob/1fd96756daf90b8d2f901fe19a168e9d11974c0b/python/cudf_polars/cudf_polars/typing/__init__.py#L22-L40

Happy to revert this if you'd prefer to adopt this recommendation from `ruff`.

Reply:
No, this one is fine.